src/pybind/mgr/dashboard/frontend/src/app/core/navigation/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -138,4 +141,14 @@ export class BreadcrumbsComponent implements OnDestroy {
 
     return of(value as T);
   }
+
+  private _getTitleFromCrumbs(crumbs: IBreadcrumb[]): string {
```

Comment:
TypeScript has private/protected/public keywords for visibility, no need to prefix with underscore.

```suggestion
  private getTitleFromCrumbs(crumbs: IBreadcrumb[]): string {
```